src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.cpp

```diff
@@ -124,8 +122,8 @@ LockFreeUserSpaceInstrumentationEventProducer& GetCaptureEventProducer() {
   return producer;
 }
 
-// Provide a thread local bool to keep track of whether the current thread is inside the payload we
-// injected. If that is the case we avoid further instrumentation.
+// Provide a thread local bool to keep track of whether the current thread is inside the payload
+// we injected. If that is the case we avoid further instrumentation.
```

Comment:
What's up with these comments? They were all already fitting in the 100-char column width.

Reply:
No idea. autoformat ruled otherwise. Fixed.